LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/authentication/KeyVaultAuthenticationClient.java

```diff
@@ -37,9 +38,9 @@
  */
 public class KeyVaultAuthenticationClient implements IAccessTokenSupplier {
 
-    private final static String CLIENT_ID = "f62c5ae3-bf3a-4af5-afa8-a68b800396e9";
-    private final static String SCOPE = "https://msidlab.com/.default";
-    private final static String CERTIFICATE_ALIAS = "LabVaultAccessCert";
+    private final static String CLIENT_ID = LabConstants.DEFAULT_LAB_CLIENT_ID;
+    private final static String SCOPE = LabConstants.DEFAULT_LAB_SCOPE;
```

Comment:
The way I understand it, there are (currently, since [my PR](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2439/files)), 3 types of auth clients in scope:
1. `KeyVaultAuthenticationClient`: fetch tokens for the KeyVault Labs API (i.e. `msidlab.com/api/LabSecret?secret={value}`
2. `LabApiAuthenticationClient`: fetch tokens for other Labs API functionality (e.g. `https://msidlab.com/api/app/{value}`
3. `KeyVaultCertificateBasedAuthenticationClient`: fetch tokens for the KeyVault API directly (not through Labs API) (e.g. `buildautomation.vault.azure.net/secrets/{value}`

My PR has introduced #3. Given this, can we rename #1 (i.e. this class) to something that makes it clearer what the difference is? To me, this current class name sounds like you're authing against the KeyVault, which you're not: you're authing against the Labs API.

Reply:
Moreover: what is the functional difference between `KeyVaultAuthenticationClient` and `LabApiAuthenticationClient`. Since they both auth against the Labs API, why do we need 2 classes?

Reply:
At the very least can you add Javadoc to these classes explaining their difference?

Reply:
Let's have a discussion when you get back, I'll need to merge this as is now but will definitely address this confusion in another PR